xslt/xml2manualcsv.xslt

```diff
@@ -38,7 +38,7 @@
           $fs)"/>
 
         <!-- Print cylinder info according to the amount of cylinders in dive -->
-        <xsl:for-each select="dive/cylinder|trip/dive/cylinder">
+        <xsl:for-each select="dive[position()=last()]/cylinder|trip[position()=last()]/dive[position()=last()]/cylinder">
```

Comment:
so if the last dive has only one cylinder, we don't get additional cylinders from an earlier dive?

Reply:
yep, that is how it goes. I think using the last dive as reference is more logical than first dive. And with XSLT it is not feasible to look for the max amount of cylinders in any of the dives to be exported (due to poor performance). So this chosen compromise appeared to make most sense to me.